spec/metrics/metrics_spec.rb

```diff
@@ -1,179 +1,159 @@
+
+    context "after_write callback" do
+      it "sets current time" do
+        Timecop.freeze(Time.local(2015))
+        dummy_metrics.write!
+        expect(dummy_metrics.time).to eq Time.local(2015)
       end
     end
   end
 
-  describe "class methods" do
+  describe "#series" do
     let(:dummy_metrics) do
-      Class.new(Influxer::Metrics) do
+      Class.new(described_class) do
         set_series :dummies
         attributes :user_id, :dummy_id
       end
     end
 
     let(:dummy_metrics_2) do
-      Class.new(Influxer::Metrics) do
+      Class.new(described_class) do
         set_series "dummy \"A\""
       end
     end
 
     let(:dummy_metrics_3) do
-      Class.new(Influxer::Metrics) do
+      Class.new(described_class) do
         set_series /^.*$/
       end
     end
 
     let(:dummy_with_2_series) do
-      Class.new(Influxer::Metrics) do
+      Class.new(described_class) do
         set_series :events, :errors
       end
     end
 
     let(:dummy_with_2_series_quoted) do
-      Class.new(Influxer::Metrics) do
+      Class.new(described_class) do
         set_series "dummy \"A\"", "dummy \"B\""
       end
     end
 
     let(:dummy_with_proc_series) do
-      Class.new(Influxer::Metrics) do
+      Class.new(described_class) do
         attributes :user_id, :test_id
         set_series ->(metrics) { "test/#{metrics.test_id}/user/#{metrics.user_id}" }
       end
     end
 
-    describe "set_series" do
-      it "should set series name from class name by default" do
-        expect(DummyMetrics.new.series).to eq "\"dummy\""
-      end
+    it "sets series name from class name by default" do
+      expect(DummyMetrics.new.series).to eq "\"dummy\""
+    end
 
-      it "should set series from subclass" do
-        expect(dummy_metrics.new.series).to eq "\"dummies\""
-      end
+    it "sets series from subclass" do
+      expect(dummy_metrics.new.series).to eq "\"dummies\""
+    end
 
-      it "should set series as regexp" do
-        expect(dummy_metrics_3.new.series).to eq '/^.*$/'
-      end
+    it "sets series as regexp" do
+      expect(dummy_metrics_3.new.series).to eq '/^.*$/'
+    end
 
-      it "should set series with quotes" do
-        expect(dummy_metrics_2.new.series).to eq "\"dummy \\\"A\\\"\""
-      end
+    it "quotes series" do
+      expect(dummy_metrics_2.new.series).to eq "\"dummy \\\"A\\\"\""
+    end
 
-      it "should set several series" do
-        expect(dummy_with_2_series.new.series).to eq "merge(\"events\",\"errors\")"
-      end
+    it "set several series" do
+      expect(dummy_with_2_series.new.series).to eq "merge(\"events\",\"errors\")"
+    end
 
-      it "should set several series with quotes" do
-        expect(dummy_with_2_series_quoted.new.series)
-          .to eq "merge(\"dummy \\\"A\\\"\",\"dummy \\\"B\\\"\")"
-      end
+    it "quotes several series" do
+      expect(dummy_with_2_series_quoted.new.series)
+        .to eq "merge(\"dummy \\\"A\\\"\",\"dummy \\\"B\\\"\")"
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.